.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # The @googleapis/yoshi-python is the default owner for changes in this repo
-*               @arithmetic1728 @sai-sunder-s @TimurSadykov @googleapis/yoshi-python
+*               @arithmetic1728 @sai-sunder-s @TimurSadykov @googleapis/yoshi-python @googleapis/googleapis-auth
```

Comment:
```suggestion
*               @googleapis/googleapis-auth @googleapis/yoshi-python
```

Reply:
Lets still keep someone directly here? Just in case the group runs into some issues.
Maybe leave Sijun and myself